src/components/sw-issue-history-card.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+  <q-card flat class="mb-2 rounded">
+    <q-card-section>
+      <div class="row justify-between items-center">
+        <div class="col-9 row items-center">
+          <q-icon
+            name="today"
+            class="col-2 q-mr-sm"
+            size="48px"
+            color="primary" />
+          <div class="col content-items">
+            <div
+              v-if="label.length > 25"
+              class="text-weight-bold"
+              style="font-size: 14px; line-height: 26px">
+              {{ label.substring(0, 25) + "..." }}
```

Comment:
creer une fonction ou 
```suggestion
              {{`${ label.substring(0, 25) }...`}}
```